celer/tests/conv_warning/visualize_logs.py

```diff
@@ -0,0 +1,62 @@
+import re
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+def parse_logs(filename: str) -> np.ndarray:
```

Comment:
the more you use complicated code, the more you risk making mistakes without noticing it. try to keep it as simple as possible